README.md

```diff
@@ -53,17 +181,219 @@ const createMyFirstNFT = async () => {
 }
 ```
 
-## Documentation
+**That being said, You can opt for the manual route which although being complex, offers more  versatility.**
+
+The Basic way automated the 3 steps (Create -> Sign => Send) assosiated with creating an NFT making it much easier to use while not allowing any customisation.
+
+The manual way provides much more versatility but is significantly more complex.Let's say for example you can batch transactions together instead of executing them one by one (covered in example section). It'll be useful if you want to simplify the process of creating a large amount of NFTs and minimize repetitive tasks like sending and signing each transaction.
+
+##### STEP 1 - Create an NFT transaction
+First of all, instead of using the `createNft()` function, you will use `createNftTx()`. And instead of creating, signing and submiting the transaction and getting the returned events, it will just create an unsigned and unsubmitted Create-NFT Transaction Hash which will be valid for the next 5 minutes.
+
+```javascript
+// Imports
+import { initializeApi } from "ternoa-js"
+import { createNftTx } from "ternoa-js/nft"
+
+const create createNFTManually = async () => {
+  try {
+    // STEP 1 : Here we create the transaction and get the transaction hash
+    const nftTxHash = await createNftTx("My first NFT", 10, undefined, false)
+
+    // Do something with the transaction hash
+    console.log(nftTxHash);
+  } catch (e) {
+    console.log(e)
+  }
+}
+```
+
+##### STEP 2 - Sign a transaction hash
+Now we have the `txHash`, we can move to the signing step. But before detailing it, it's good to know that "signing" can be directely embed in the submit function. It means that depending on the submit function you are using (see the last step below), signing your tx hash before submit might not be necessary. In case you sign manually the tx hash, you will receive a hex value of the signed transaction ready to be send. The `signTxHex()` function expect **a keyring** that will sign the transaction and the **transaction hash to be signed**.
```

Comment:
Typo error: 'that "signing" can be directely embed' => ''that "signing" can be directly embed'

Reply:
Done